api/src/opentrons/protocol_api_experimental/protocol_context.py

```diff
@@ -225,5 +225,12 @@ def pause(self, msg: Optional[str] = None) -> None:
         """
         self._engine_client.pause(message=msg)
 
+    def set_rail_lights(self, on: bool) -> None:
+        """Controls the robot rail lights.
+
+        :param bool on: If true, turn on rail lights; otherwise, turn off.
+        """
```

Comment:
```suggestion
        """Controls the robot rail lights.

        Args:
            on: If true, turn on rail lights; otherwise, turn off.
        """
```

We've been moving towards [Google-style docstrings](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html), for syntactic simplicity. (Not that it really matters since this isn't rendered anywhere.)